src/AspNet.Security.OAuth.Extensions/ClaimsExtensions.cs

```diff
@@ -13,6 +13,11 @@ public static class ClaimsExtensions
     {
         public static ClaimsIdentity AddOptionalClaim(this ClaimsIdentity identity, string type, string value, string issuer)
         {
+            return AddOptionalClaim(identity, type, value, ClaimValueTypes.String, issuer);
+        }
+
+        public static ClaimsIdentity AddOptionalClaim(this ClaimsIdentity identity, string type, string value, string types, string issuer)
```

Comment:
Or you can remove the existing `type` parameter to `name` and `types` to `types` if you prefer :+1: